apps/heft/src/schemas/typescript.schema.json

```diff
@@ -38,6 +38,11 @@
       }
     },
 
+    "emitFolderPathForJest": {
```

Comment:
We could probably infer this easily.

Reply:
I'm not so sure about that.  Consider that:
- The main `lib` folder is specified by **tsconfig.json**, often via an `"extends"` chain
- There can be more than one output module that is `commonjs` type, and thus usable by Jest

It's probably less magical for `emitFolderPathForJest` to be an explicit setting, in cases where it is not simply the `lib` default name.